support-frontend/assets/components/marketingConsentNew/marketingConsent.scss

```
@@ -1,21 +1,18 @@
-// -----  gu-sass ----- //	
+// -----  gu-sass ----- //
 @import '~stylesheets/gu-sass/gu-sass';
 
 .component-marketing-consent {
   font-weight: 300;
   margin: 0 -10px;
-  padding: 0 10px;
+  padding: 0 10px ($gu-v-spacing * 2);
   .svg-newsletters {
     width: 100%;
   }
 }
 
 .component-marketing-consent-confirmation {
   font-family: $gu-text-sans-web;
-  font-size: 12px;
-}
-
-.component-marketing-consent-confirmation__message {
-  vertical-align: top;
-  padding-left: 5px;
+  font-size: 14px;
```

```suggestion
  @include gu-fontset-explainer;
```

You can use this mixin to get text-sans at 14px